README.md

```diff
@@ -226,11 +227,25 @@ A job scheduler is a computer application for controlling unattended background
 
 Note that the non enterprise version of [Sidekiq](https://github.com/mperham/sidekiq) doesn't do scheduling by default, it only executes jobs
 
+## Improving this template
 
+We have a script which will build various configurations of this template into the `template-test/dummy` directory.
 
+```bash
+$ ./bin/build -h
+Usage: ./bin/template-run-ci [options]
+    -d, --drop-dbs [FLAG]            Drop databases before running
+    -h, --help                       Prints this help
+    -l, --list-for-ci                Output the commands that should be embedded into Github Actions yaml
+    -v, --variants VARIANT_NAMES     Comma separated list of variants to build and test
 
+# setup all variants of the template into template-test/dummy - read the build
+# script code for details of each variant
+$ ./bin/build
 
+# setup the basic variant only
+$ ./bin/build -v basic
 
-
-
-
+# generate some output for building the variants suitable for pasting into ./github/ci.yml
+$ ./bin/build --list-for-ci
```

Comment:
This is particularly handy to avoid having to puzzle out making the generator input manually